### PR TITLE
Add deployment script

### PR DESCRIPTION
**Pull request by cawabunga**
Part of #19

Presently it is impossible to use `hardhat-upgrades` to deploy contracts, because they don't inherit `Initializable` contract.
I almost managed to update them, but struggled to fix `SCTERC20Token` contract. 

Also `sSolidERC20` and `SolidERC20` are not valid, so in order to run `npx hardhat run scripts/deploy.js` they need to be removed.

### CODE REVIEW

**Review by jeffprestes — APPROVED** [review]

**Comment by jeffprestes** [comment]
Actually, you did correctly because those contracts were not designed to be upgradable.

**Comment by cawabunga** [comment]
What do you think about separating the deployer and the governer accounts? Because now they are the same, who deploys becomes the governer.

**Review by jeffprestes — COMMENTED** [review]
The source looks good. However, when I had tried to run it I got this error:

```bash
$ npm i

added 322 packages, and audited 323 packages in 35s

66 packages are looking for funding
  run `npm fund` for details

4 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
~/projetos/go/src/github.com/solid-world/solid-world-dao-contracts (19-add-deployment-script)$ npx hardhat deploy --network mumbai
Governor: 0x8b3a08b22d25c60e4b2bfd984e331568eca4c299
Guardian: 0x94cd0f84fec287f2426e90f0d6653ba8fa29bd8e
  Policy: 0x513906d9b238955b7e4a499ad98e0b90f9503eb4
   Vault: 0x1570bcaebf4a6184814bb8bde72c9b96f3d37525


Error HH700: Artifact for contract "SolidDaoManagement" not found.
```

File: README.md [hunk 1]
```diff
@@ -41,4 +41,27 @@ Source: https://github.com/solid-world/solid-world-dao-contracts/blob/f72db030ba
...
+- Create `.env` (see `.env.example`): add deployer's creds, guardian, policy and vault addresses
+- Run the following command:
+
+```sh
+npx hardhat deploy --network {rinkeby | ropsten | main | mumbai}
+```
+
+For verification the contract after deployment run:
+```sh
+npx hardhat verify --network {rinkeby | ropsten | main | mumbai} DEPLOYED_DAO_CONTRACT_ADDRESS GOVERNER_ADDRESS GUARDIAN_ADDRESS POLICY_ADDRESS VAULT_ADDRESS
```
Comment (jeffprestes) [inline README.md:59]: Here the correctly might be: npx hardhat verify --network {rinkeby | ropsten | main | mumbai} DEPLOYED_CONTRACT_ADDRESS PARAM1 PARAM_N...

File: README.md [hunk 2]
```diff
@@ -41,4 +41,27 @@ Source: https://github.com/solid-world/solid-world-dao-contracts/blob/f72db030ba
...
+
+### How to deploy to network
+
+- Add fund to deployer account on target network
+- Create `secrets.json` (see `secrets.json.example`): add Etherscan API key and Infura API key
+- Create `.env` (see `.env.example`): add deployer's creds, guardian, policy and vault addresses
+- Run the following command:
+
+```sh
+npx hardhat deploy --network {rinkeby | ropsten | main | mumbai}
```
Comment (jeffprestes) [inline README.md:54]: Before to call the deploy it would be important to add npx hardhat compile too

**Review by jeffprestes — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cawabunga) [reply README.md:54]: I added compilation directly to deploy task.

**Review by jeffprestes — APPROVED** [review]

**Pull request merged**